tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -1036,7 +1056,17 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 			framework.ExpectNoError(err, "wait for TidbCluster timeout: %v", targetTc)
 
 			drainerConfig := &tests.DrainerConfig{
-				DrainerName:       "tls-drainer",
+				// Note: DrainerName muse be tcName
+				// oa.DeployDrainer will use DrainerName as release name to run "helm install..."
+				// in InstallTiDBCertificates, we use `tidbComponentsCertificatesTmpl` to render the certs:
+				// ...
+				// dnsNames:
+				// - "*.{{ .ClusterName }}-{{ .ClusterName }}-drainer"
+				// - "*.{{ .ClusterName }}-{{ .ClusterName }}-drainer.{{ .Namespace }}"
+				// - "*.{{ .ClusterName }}-{{ .ClusterName }}-drainer.{{ .Namespace }}.svc"
+				// ...
+				// we will drainer refer to: https://docs.pingcap.com/zh/tidb-in-kubernetes/drainer/enable-tls-between-components
```

Comment:
I don't know what does `we will drainer refer to` mean

Reply:
updated comment